Assets/Scripts/Ffb/Dto/Reports/Injury.cs

```diff
@@ -17,7 +18,7 @@ public class Injury : Report
         public int? injury;
         public int? injuryDecay;
         public string attackerId;
-        public string[] armorModifiers;
+        public List<FFBEnumeration> armorModifiers;
         public string[] injuryModifiers;
```

Comment:
injuryModifiers should also be a List<FFBEnumeration>, may as well change it while you're at it.

Reply:
This will come in the next PR (which I am currently working on)